examples/antarctic_slope_current_model.jl

```diff
@@ -0,0 +1,154 @@
+sponge_layers      = (south_sponge_layer, north_sponge_layer)
+# TODO: compose north_mask and south_mask together into one sponge layer, OR compose north/south sponge layers
+
+
+# Boundary Conditions:
+no_slip_bc   = ValueBoundaryCondition(0.0)
+free_slip_bc = FluxBoundaryCondition(nothing)
+
+free_slip_surface_bcs = FieldBoundaryConditions(no_slip_bc, top=FluxBoundaryCondition(nothing))
+no_slip_field_bcs     = FieldBoundaryConditions(no_slip_bc)
+
+# Buoyancy Equations of State - we want high order polynomials, so we'll use TEOS-10
+eos = TEOS10EquationOfState()
+
+# Coriolis Effect, using basic f-plane with precribed reference Coriolis parameter
+coriolis = FPlane(f=-1.3e14)
+
+# Diffusivities as part of closure
+# TODO: make sure this works for biharmonic diffusivities as the horizontal, 
+horizontal_closure = HorizontalScalarDiffusivity(ν=0.1, κ=0.1)
+vertical_closure   = VerticalScalarDiffusivity(ν=3e-4, κ=1e-5)
+
+# Assuming no particles or biogeochemistry
+model = HydrostaticFreeSurfaceModel(; grid,
+                                         clock = Clock{eltype(grid)}(0, 0, 1),
+                            momentum_advection = CenteredSecondOrder(),
+                              tracer_advection = CenteredSecondOrder(),
+                                      buoyancy = SeawaterBuoyancy(equation_of_state=eos),
+                                      coriolis = coriolis,
+                                  free_surface = ImplicitFreeSurface(gravitational_acceleration=g_Earth),
+                                       forcing = (u=sponge_layers, v=sponge_layers, w=sponge_layers, T=sponge_layers, S=sponge_layers), # NamedTuple()
+                                       closure = (horizontal_closure, vertical_closure),
+                           boundary_conditions = (u=free_slip_surface_bcs, v=free_slip_surface_bcs, w=no_slip_field_bcs), # NamedTuple(),
+                                       tracers = (:T, :S),
+                                    velocities = nothing,
+                                      pressure = nothing,
+                            diffusivity_fields = nothing,
+                              #auxiliary_fields = nothing, # NamedTuple(),
+)
+
+println(model)
+
+println("u boundary conditions:")
+println(model.velocities.u.boundary_conditions)
+println("v boundary conditions:")
+println(model.velocities.v.boundary_conditions)
+println("w boundary conditions:")
+println(model.velocities.w.boundary_conditions)
+
+#
+# Now create a simulation and run the model
+#
+simulation = Simulation(model; Δt=100.0, stop_time=600minutes)
+
+# Create a NamedTuple
+
+filename = "asc_model_run"
+
+simulation.output_writers[:slices] =
+    JLD2OutputWriter(model, merge(model.velocities, model.tracers),
+                     filename = filename * ".jld2",
+                     indices = (:, grid.Ny/2, :),
+                     schedule = TimeInterval(1minute),
+                     overwrite_existing = true)
```

Comment:
The most interesting output will be a slice at the surface, eg using

```julia
indices = (:, :, grid.Nz)
```

so make sure to always output that